contracts/LiquidityGauge.vy

```diff
@@ -90,6 +93,7 @@ MINTER: constant(address) = 0xd061D61a4d941c39E5453435B6345Dc261C2fcE0
 CRV: constant(address) = 0xD533a949740bb3306d119CC777fa900bA034cd52
 VOTING_ESCROW: constant(address) = 0x5f3b5DfEb7B28CDbD7FAba78963EE202a494e2A2
 GAUGE_CONTROLLER: constant(address) = 0x2F50D538606Fa9EDD2B11E2446BEb18C9D5846bB
+VEBOOST_PROXY: constant(address) = 0x0000000000000000000000000000000000000000
```

Comment:
This needs to be updated to the correct veboost proxy address

Reply:
[`0x8E0c00ed546602fD9927DF742bbAbF726D5B0d16`](https://etherscan.io/address/0x8e0c00ed546602fd9927df742bbabf726d5b0d16#code)